docs/source/training_tutorials/finetune_llm.mdx

```diff
@@ -266,8 +263,7 @@ The compilation command simply consists in calling your script as an input to th
 
 ```bash
 MALLOC_ARENA_MAX=64 XLA_USE_BF16=1 neuron_parallel_compile torchrun --nproc_per_node=32 finetune_llm.py \
- --model_id {model_id} \
- --dataset_path {dataset_path} \
+ --model_id meta-llama/Llama-2-7b-hf \
```

Comment:
Shall we use the same checkpoint in this tutorial, especially it's titled with llama3 8B.

Reply:
Good catch.